pandas-stubs/io/pytables.pyi

```diff
@@ -1,76 +1,102 @@
 from typing import (
     Any,
-    Hashable,
+    Literal,
+    overload,
 )
 
 import numpy as np
+from pandas import (
+    DataFrame,
+    Index,
+    Series,
+)
 from pandas.core.computation.pytables import PyTablesExpr
-from pandas.core.frame import DataFrame
 from pandas.core.generic import NDFrame
-from pandas.core.indexes.base import Index
-from pandas.core.indexes.multi import MultiIndex
-from pandas.core.series import Series
 
 from pandas._typing import (
-    ArrayLike,
-    FilePathOrBuffer,
+    FilePath,
+    HashableT,
 )
 
-from pandas.core.dtypes.generic import ABCExtensionArray
-
-# pytables may not be installed so create them as dummy classes
-class Col: ...
-class Node: ...
-
 Term = PyTablesExpr
 
 class PossibleDataLossError(Exception): ...
 class ClosedFileError(Exception): ...
 class IncompatibilityWarning(Warning): ...
-
-incompatibility_doc: str
-
 class AttributeConflictWarning(Warning): ...
-
-attribute_conflict_doc: str
-
 class DuplicateWarning(Warning): ...
 
-duplicate_doc: str
-performance_doc: str
-dropna_doc: str
-format_doc: str
-
-def to_hdf(
-    path_or_buf: FilePathOrBuffer,
-    key: str,
-    value: NDFrame,
-    mode: str = ...,
-    complevel: int | None = ...,
-    complib: str | None = ...,
-    append: bool = ...,
-    format: str | None = ...,
-    index: bool = ...,
-    min_itemsize: int | dict[str, int] | None = ...,
-    nan_rep=...,
-    dropna: bool | None = ...,
-    data_columns: list[str] | None = ...,
-    errors: str = ...,
-    encoding: str = ...,
-): ...
+@overload
 def read_hdf(
-    path_or_buf: FilePathOrBuffer,
-    key=...,
-    mode: str = ...,
-    errors: str = ...,
-    where: list[Any] | None = ...,
+    path_or_buf: FilePath | HDFStore,
+    key: Any | None = ...,
+    mode: Literal["r", "r+", "a"] = ...,
+    errors: Literal[
+        "strict",
+        "ignore",
+        "replace",
+        "surrogateescape",
+        "xmlcharrefreplace",
+        "backslashreplace",
+        "namereplace",
+    ]
+    | None = ...,
```

Comment:
Not sure that `None` is acceptable for `errors` argument.

Reply:
Fixed.